packages/gatsby-plugin-feed/src/internals.js

```diff
@@ -29,11 +29,13 @@ export const defaultOptions = {
   setup: ({
     query: {
       site: { siteMetadata },
-      ...rest
     },
+    generator,
```

Comment:
This doesn't seem necessary! 

```suggestion

```

Reply:
This is necessary; otherwise, the default `generator` value "GatsbyJs" will be ignored.

Reply:
Functionally it will behave the same now that you've fixed the issue with `...rest` being in the wrong place - I've checked that all your tests still pass just to make sure. Maybe it was only necessary before you fixed the primary issue?

Reply:
@davidbailey00 Interestingly enough, that is true. It's funny because that only showed up on the tests _after_ I had already fixed the primary issue. Oh well, who knows why, I removed it.